web/annotation_editor_layer_builder.css

```
@@ -1032,6 +1032,11 @@
 
           &:has(.dropdown:not(.hidden)) {
             background-color: var(--editor-toolbar-hover-bg-color);
+
+            &::after {
+              transform-origin: center;
+              rotate: 180deg;
```

Looking at how we flip some icons in the viewer main-toolbar (in that case for RTL locales), could we make this a one-liner instead?
```suggestion
              transform: scaleY(-1);
```

---

Did this not work?

---

Did this suggestion not work out, or did you just choose not to change it?

Note that I'm only asking since the comment was marked as resolved, without any code-changes being made or a comment left about this :-)